alert/alert.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />

```

Comment:
You can always include your name in the project meta tag just to give a personalised touch.

Reply:
Will do that, thanks for the suggestion.